cmd/whoisparser/main.go

```diff
@@ -0,0 +1,78 @@
+package main
```

Comment:
Please remove the `cmd` directory from the git index as it is used only by devs to test the library and is not required to use this lib as a dependency.

Perhaps it should be better to add `cmd` dir to .gitignore.